WORKSPACE

```diff
@@ -83,6 +83,28 @@ http_archive(
     ],
 )
 
+#### Building boring ssl
+# Fetching boringssl within the flatbuffers repository, to patch the issue
+# of not being able to upgrade to Xcode 14.3 due to buildkite throwing errors
+# which was patched in the following below.
+# https://github.com/google/flatbuffers/commit/67eb95de9281087ccbba9aafd6e8ab1958d12045
+# The patch was copied from the following comment on the same issue within tensorflow
+# and fixed to adapt the already existing patch for boringssl.
+# https://github.com/tensorflow/tensorflow/issues/60191#issuecomment-1496073147
```

Comment:
Adds a description to what the issue was, and where the issue was addressed before and where is the actual patch coming from